### PR TITLE
chore(tests) switch from quintush to helm-unittest

### PR DESCRIPTION
**Pull request by dduportal**
This PR fixes the builds failure, during the unit tests steps, with the `unknown flag: --helm3` error (https://github.com/jenkins-infra/helm-charts/actions/runs/4675170591/jobs/8280015667?pr=480).

It appears that `quintush` has switched back (or was granted maintenership) on https://github.com/helm-unittest/helm-unittest as per https://github.com/quintush/helm-unittest/issues/124.

A commit was added to update the YAML manifest syntax to fulfill the new JsonPath module introduced in https://github.com/helm-unittest/helm-unittest/releases/tag/v0.3.0

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -30,14 +30,9 @@ jobs:
       - name: install helm unittests
         run: |
           helm env
-          # Repeat 2 times to fight against network errors in  GHA runners
-          # Always return true
-          helm plugin install https://github.com/quintush/helm-unittest \
-            || helm plugin install https://github.com/quintush/helm-unittest \
-            || true
```
Comment (lemeurherve) [inline .github/workflows/test.yml:37]: So we're confident enough in GitHub CDN now? ^^
I would have done that in a separate PR ;)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -30,14 +30,9 @@ jobs:
...
-          # Fail if not installed
+          helm plugin install https://github.com/quintush/helm-unittest
           helm plugin list | grep unittest
```
Comment (lemeurherve) [inline .github/workflows/test.yml:34]:
```suggestion
          # Fail if not installed
          helm plugin list | grep unittest
```
Restore comment

**Review by lemeurherve — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (dduportal) [reply .github/workflows/test.yml:37]: make sense, I was a bit enthusatsic :D

File: .github/workflows/test.yml (hunk 2 above)
Comment (dduportal) [reply .github/workflows/test.yml:41]: https://github.com/jenkins-infra/helm-charts/pull/481/commits/dea3d318d6cbf16c42161decbfc50f7182daf768

File: .github/workflows/test.yml (hunk 1 above)
Comment (dduportal) [reply .github/workflows/test.yml]: https://github.com/jenkins-infra/helm-charts/pull/481/commits/dea3d318d6cbf16c42161decbfc50f7182daf768

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -28,16 +28,18 @@ jobs:
         with:
           version: "${{ steps.tools-versions.outputs.CURRENT_HELM_VERSION }}"
       - name: install helm unittests
+        env:
+          HELM_UNITTESTS_URL: https://github.com/helm-unittest/helm-unittest
```
Comment (lemeurherve) [inline .github/workflows/test.yml:32]:
```suggestion
```
What are the chances this repo move again? This env is overkill IMO

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -28,16 +28,18 @@ jobs:
...
         run: |
           helm env
           # Repeat 2 times to fight against network errors in  GHA runners
           # Always return true
-          helm plugin install https://github.com/quintush/helm-unittest \
-            || helm plugin install https://github.com/quintush/helm-unittest \
+          helm plugin install "$HELM_UNITTESTS_URL" \
+            || helm plugin install "$HELM_UNITTESTS_URL" \
```
Comment (lemeurherve) [inline .github/workflows/test.yml:38]:
```suggestion
          helm plugin install https://github.com/helm-unittest/helm-unittest \
            || helm plugin install https://github.com/helm-unittest/helm-unittest \
```
See above
